blockbin/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
We already have a package.json file: https://github.com/ArnaudBrousseau/blockbin/blob/master/package.json -- can you merge the latest master?

Reply:
Let me try and merge this, see what happens! GH says your branch will merge fine.